nomad/client_csi_endpoint.go

```diff
@@ -41,7 +40,8 @@ func (a *ClientCSI) ControllerAttachVolume(args *cstructs.ClientCSIControllerAtt
 			return nil
 		}
 		if a.isRetryable(err) {
-			a.logger.Debug("failed to reach controller on client %q: %v", clientID, err)
+			a.logger.Debug("failed to reach controller on client",
```

Comment:
Interesting, I guess hclog vet didn't catch this since its balanced in terms of k/v pairs but incorrect :/ https://github.com/hashicorp/nomad/pull/8328

Reply:
Yeah it should probably be looking out for format directives too.